tracer/build/_build/Build.Profiler.Steps.cs

```diff
@@ -166,14 +168,27 @@ partial class Build
                 var dest = MonitoringHomeDirectory / arch / file;
                 CopyFile(source, dest, FileExistsPolicy.Overwrite);
             }
+
+            var testSource = ProfilerOutputDirectory / "bin" / "Datadog.Profiler.Native.Tests" / "Datadog.Profiler.Native.Tests";
+            var testDest = SharedDirectory / "bin" / "test";
+
+            CopyFileToDirectory(testSource, testDest, FileExistsPolicy.Overwrite);
+
+            testSource = ProfilerOutputDirectory / "bin" / "Datadog.Linux.ApiWrapper.Tests" / "Datadog.Linux.ApiWrapper.Tests";
+            CopyFileToDirectory(testSource, testDest, FileExistsPolicy.Overwrite);
+
+            // To unit tests the wrapper library too
+            testSource = sourceDir / LinuxApiWrapperLibrary;
+            CopyFileToDirectory(testSource, testDest, FileExistsPolicy.Overwrite);
```

Comment:
I feel like this should be a separate target tbh 🤔 The "Publish" stages are all about creating the actual outputs, rather than the tests I think 🤔